### DISCUSSION TITLE
problem with Series: Intro to GitHub Flow.

### DISCUSSION THREAD

**Discussion by jokaperes** [community/community#67779] [GitHub Education]
### Hi everyone,

I need help. Can i refactor or re-do this assignment?? Its not checking complete.
https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-jokaperes, this is the repository its giving bots error.

**Comment by noahjgreer** [comment]
Same problem here! Did you figure out how to restart/reset the repository to start over?

**Comment by tugrul-kok** [comment]
I am having the same problem. Did you find any solution?

**Comment by RuchitJagodara** [comment]
I am also experiencing a similar problem; it is stating that I have to navigate to the new repository. However, no repositories have been created on my account. What should I do? I tried to create a new branch on an existing repo, but it is not working.

**Reply by tugrul-kok** [reply]
Here is your repo: https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-RuchitJagodara

**Reply by RuchitJagodara** [reply]
Thank you for your help.

**Comment by celesteamador** [comment]
I have the same problem

**Comment by Bergbok** [comment]
Go to https://github.com/settings/repositories and click Leave

**Reply by eltithecoder** [reply]
This doesn't fix the problem

**Comment by utkarshgayguwal** [comment]
What i need to done for completing assingment on github flow?
